media/css/firefox/developer-quantum.scss

```diff
@@ -0,0 +1,598 @@
+            background: $color-dev-mediumblue linear-gradient(120deg, $color-dev-lightblue 0, $color-dev-mediumblue 100%) no-repeat;
+            border-radius: 100px;
+            border: 0;
+            margin-bottom: 10px;
+            padding: 1em 40px;
+
+            strong:before {
+                @include background-size(12px 28px);
+                background-image: url('/media/img/hubs/down-arrow-light-sprite.png');
+                background-position: left top;
+                content: '';
+                display: block;
+                float: left;
+                height: 14px;
+                margin: .35em 10px 0 0;
+                width: 12px;
+            }
+        }
+
+        .fx-privacy-link {
+            @include font-size-small;
+
+            a:link,
+            a:visited {
+                text-decoration: none;
+                color: $color-link;
+
+                &:hover,
+                &:focus,
+                &:active {
+                    text-decoration: underline;
+                }
+            }
+        }
+    }
+
+}
+
+
+//*------------------------------------------------------------------*/
+// Intro
+.section-intro {
+    background-color: #fff;
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    overflow: hidden;
+    text-align: center;
+
+    .page-title {
+        @include font-size-level2;
+        text-transform: uppercase;
+        margin: 0 auto 1em;
+    }
+
+    .tagline {
+        max-width: $width-tablet;
+        text-align: center;
+        margin: 0 auto 1.5em;
+    }
+
+    .download-button {
+        margin: 40px auto 20px;
+    }
+
+    .feedback-note {
+        @include font-size-small;
+        margin: 0 auto 2em;
+        max-width: $width-phone;
+        color: #aaa;
+
+        a:link,
+        a:visited {
+            @include trailing-arrow;
+            text-decoration: none;
+
+            &:hover,
+            &:focus,
+            &:active {
+                text-decoration: underline;
+            }
+        }
+    }
+
+    .hero-image {
+        display: none;
+    }
+
+    @media #{$mq-tablet} {
+        @include background-size(1700px 450px);
+        background-image: url('/media/img/firefox/products/developer/quantum/hero-background.png');
+        padding-top: 40px;
+
+        .hero-image {
+            display: block;
+            margin: 40px auto -60px;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        @include at2x('/media/img/firefox/products/developer/quantum/hero-background.png', 3400px, 900px);
+
+        .hero-image {
+            margin-top: 60px;
+        }
+    }
+
+    @media #{$mq-desktop-wide} {
+        .hero-image {
+            margin-bottom: -80px;
+        }
+    }
+}
+
+
+//*------------------------------------------------------------------*/
+// Performance
+.section-performance {
+    .section-title {
+        color: #3078fb;
+
+        span {
+            border-color: #3078fb;
```

Comment:
Wonder if it is worth moving this to a variable as well?

Reply:
Good catch, these should be $color-dev-mediumblue.